share/spack/gitlab/cloud_pipelines/stacks/developer_tools/spack.yaml

```diff
@@ -0,0 +1,70 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=x86_64_v3
+  concretizer:
+    reuse: false
```

Comment:
you can remove the `reuse: false` from here. I think it is better to not have this specified in stacks to avoid it being turned on by accident.

Reply:
Sure, it being on was a problem and why that's there.

```suggestion
```